P9Operators/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿Console.WriteLine("Give me a number of seconds");
+string input = Console.ReadLine();
+
+float floatValue = float.Parse(input);
+
+float total = floatValue / (60 * 60 * 24);
```

Comment:
Instead of these magic numbers, consider using variables:

```csharp
int minuteToSeconds = 60;
int hourToSeconds = 60*minuteToSeconds;
int dayToSeconds = 24*hourToSeconds;

doubleTotalDaysFraction = (double)totalSeconds / dayToSeconds;
```

Reply:
Hi, I've changed the code, can you check the new solution?